tests/test_smartdataframe.py

```diff
@@ -251,9 +265,8 @@ def analyze_data(dfs: list[pd.DataFrame]) -> dict:
     2. Process: Manipulating data for analysis (grouping, filtering, aggregating, etc.)
     3. Analyze: Conducting the actual analysis (if the user asks to plot a chart save it to an image in exports/charts/temp_chart.png and do not show the chart.)
     4. Output: return a dictionary of:
-    - type (must be "number")
-    - value (must be a number)
-    Example output: { "type": "text", "value": "The average loan amount is $15,000." }
+    {output_type_hint}
+    Example output: {{ "type": "text", "value": "The average loan amount is $15,000." }}
```

Comment:
Maybe even the example output might change based on the type so that we don't risk hallucinating the LLM?

Reply:
Positive. Really great notice, this might have turned a dull confusion for the AI. I must just lost sight of that
___
Done